src/components/extensive/Tables/DisturbancesTablePD.tsx

```diff
@@ -1,9 +1,11 @@
 import Table from "@/components/elements/Table/Table";
 import { VARIANT_TABLE_TREE_SPECIES } from "@/components/elements/Table/TableVariants";
-import { useGetV2DisturbancesENTITYUUID } from "@/generated/apiComponents";
+import { useDisturbances } from "@/connections/EntityAssocation";
+
+import { DemographicEntity } from "../DemographicsCollapseGrid/types";
 
 export interface DisturbancesTableProps {
-  modelName: string;
+  modelName: DemographicEntity;
```

Comment:
Ah, now I see why you modified that type. This should get its own type. Demographics and Disturbances are different models and can be attached to different entities.